examples/td3/config.yaml

```diff
@@ -1,47 +1,49 @@
-# Environment
+# task and env
 env:
   name: HalfCheetah-v3
   task: ""
-  exp_name: "HalfCheetah-TD3"
+  exp_name: "HalfCheetah-TD3-ICLR"
   library: gym
   frame_skip: 1
   seed: 42
 
-# Collection
+# collector
 collector:
-  total_frames: 1000000
-  init_random_frames: 10000
+  total_frames: 3000000
+  init_random_frames: 25_000
   init_env_steps: 1000
-  frames_per_batch: 1000
+  frames_per_batch: 1
```

Comment:
this is going to be super inefficient, do we have to stick to that?
can we obtain the same perf with something faster?

Reply:
Performance might be a bit less stable, I'm testing it right now.